deploy/examples/export-external-cluster.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/bash
+###############################
+
+USAGE=$(cat <<-'EOF'
```

Comment:
My plan is to work on updating the documentation once this pull request is accepted.  Is that plan acceptable or should the documentation updates also be part of this pull request?  I didn't want to put a lot of work into the documentation if no one was interested in my script.

Reply:
Stepping back for a minute, this script seems better outside the rook repo. It is pulling specific releases of the rook manifests and creating them in a way that the Rook scripts haven't done previously. Any scripts in the rook repo should just be able to reference scripts or example manifests in the same directory instead of curling them from a release. Even so, If improvements to external cluster config are going to be integrated with Rook, I'm envisioning additions to the existing export or import scripts. For example, could a new flag be added to the export script for saving the yaml resources, and a flag added to the import script for creating the yaml resources?

Reply:
@travisn the original issue requested just what you have asked for, to enhance the existing script to write out the files rather than just use kube apply.  I just thought this script might be useful as it works, until something better comes along.  If appropriate, I can cancel this pull request and save a copy of my script to the issue for people to use until someone creates a better solution that meets rook's standards.  Shall I do this?  I don't really want to rewrite the import script to export files ... this script was just something I wrote in a few minutes.  The solution you propose would be a pretty big change and I wouldn't feel confident writing it, I suspect it would essentially require a complete rewrite of the import script.